### PR TITLE
Add git autoupdate config in AnythingSlider

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
You should confirm the structure under `js` folder ...
https://github.com/CSS-Tricks/AnythingSlider/tree/v1.9.4/js

**Comment by LeaYeh** [comment]
我好像看成別的檔案了XD
只是我真的不清楚 "*" 什麼時候不能用

**Comment by PeterDaveHello** [comment]
You dropped css files?

**Comment by LeaYeh** [comment]
you mean this?

File: ajax/libs/anythingslider/package.json [hunk 1]
```diff
@@ -20,5 +20,15 @@
...
       "url": "https://github.com/CSS-Tricks/AnythingSlider"
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/CSS-Tricks/AnythingSlider.git",
+    "basePath": "",
+    "files": [
+      "js/*.js",
```
Comment (PeterDaveHello) [inline ajax/libs/anythingslider/package.json]: Please check the added version and try to make the file structure consistent.

**Comment by LeaYeh** [comment]
check it.

**Comment by PeterDaveHello** [comment]
Looks better now, and would you please also add themes in css? Thanks.

**Comment by LeaYeh** [comment]
ok

**Comment by PeterDaveHello** [comment]
Nice.

**Comment by telkomsel13** [comment]
#133

**Pull request merged**